apps/framework-cli/src/utilities/git.rs

```diff
@@ -61,18 +61,23 @@ pub fn dump_old_version_schema(
     commit_hash: String,
     dest: &Path,
 ) -> Result<(), Error> {
-    let repo = Repository::open(project.project_location.clone())?;
+    let repo = Repository::discover(project.project_location.clone())?;
 
-    let path = project.schemas_dir();
-    let path = path.strip_prefix(&project.project_location).unwrap();
+    let schema_dir = project.schemas_dir();
+    let schema_relative_path_from_repo_root = schema_dir
+        .strip_prefix(repo.path().parent().unwrap())
```

Comment:
Are you confident with the `unwrap` here? looks like the method returns a `Result` so you could use `?` instead